ui/src/app/components/new-user-satisfaction-survey-modal.spec.tsx

```diff
@@ -47,81 +57,100 @@ describe(NewUserSatisfactionSurveyModal.name, () => {
     setValidNewUserSatisfactionSurveyData();
   });
 
-  const createShallowWrapper = ({
+  const renderTestComponent = ({
     onCancel = () => {},
     onSubmitSuccess = () => {},
     createSurveyApiCall = () => Promise.resolve(),
   } = {}) => {
-    return shallow(
+    return render(
       <NewUserSatisfactionSurveyModal
-        onCancel={onCancel}
-        onSubmitSuccess={onSubmitSuccess}
-        createSurveyApiCall={createSurveyApiCall}
+        {...{ onCancel, onSubmitSuccess, createSurveyApiCall }}
       />
     );
   };
 
-  const findSubmitButton = (wrapper: ShallowWrapper) => {
-    return wrapper.find({ children: 'Submit' });
+  const findSubmitButton = (component: RenderResult) => {
```

Comment:
This might not be necessary, but what do you think about renaming this to getSubmitButton, so it matches the RTL function being used?